subsys/jwt/jwt.c

```diff
@@ -229,13 +230,15 @@ int jwt_sign(struct jwt_builder *builder,
 			      sig, sig_len, &sig_len,
```

Comment:
Speaking of error handling, the call to `mbedtls_sha256` does not check the return value.
@Yago-Caetano Could you check that?